modules/tinymce/src/plugins/quickbars/test/ts/browser/KeyboardShortcutContextualToolBarTest.ts

```diff
@@ -0,0 +1,30 @@
+import { FocusTools, Keys } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { SugarDocument } from '@ephox/sugar';
+import { TinyHooks, TinySelections, TinyContentActions, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/quickbars/Plugin';
+
+describe('browser.tinymce.plugins.quickbars.KeyboardShortcutContextualToolBarTest', () => {
```

Comment:
```suggestion
describe('browser.tinymce.plugins.quickbars.KeyboardShortcutContextualToolbarTest', () => {
```
Toolbar is one word here, so the `B` shouldn't be captialised.